packages/gatsby/src/services/post-bootstrap.ts

```diff
@@ -0,0 +1,23 @@
+import reporter from "gatsby-cli/lib/reporter"
+import { IBuildContext } from "./"
+import { emitter } from "../redux"
+import apiRunnerNode from "../utils/api-runner-node"
+
+export async function postBootstrap({
+  parentSpan,
+}: Partial<IBuildContext>): Promise<void> {
+  const activity = reporter.activityTimer(`onPostBootstrap`, {
+    parentSpan,
+  })
+  activity.start()
+  await apiRunnerNode(`onPostBootstrap`, { parentSpan: activity.span })
+  activity.end()
+
+  reporter.log(``)
+  reporter.info(`bootstrap finished - ${process.uptime().toFixed(3)}s`)
+  reporter.log(``)
```

Comment:
I guess this wasn't your change but can't this pre/append newlines to the middle one instead?

Reply:
Yeah, this was how it was before. Following the principle of fewest irrelevant changes, I'd not refactored anything like this.

Reply:
Done